src/api.js

```diff
@@ -0,0 +1,14 @@
+const fetchData = (param) => {
```

Comment:
You may want to add in a catch here for any console errors but I think that is up to what makes the most sense for you when you get to posting.